components/camel-aws-ddb/src/main/java/org/apache/camel/component/aws/ddb/DdbComponentVerifierExtension.java

```diff
@@ -75,11 +89,11 @@ protected Result verifyConnectivity(Map<String, Object> parameters) {
             client.listTables();
         } catch (SdkClientException e) {
             ResultErrorBuilder errorBuilder = ResultErrorBuilder.withCodeAndDescription(VerificationError.StandardCode.AUTHENTICATION, e.getMessage())
-                .detail("aws_ddb_exception_message", e.getMessage()).detail(VerificationError.ExceptionAttribute.EXCEPTION_CLASS, e.getClass().getName())
-                .detail(VerificationError.ExceptionAttribute.EXCEPTION_INSTANCE, e);
+                    .detail("aws_ddb_exception_message", e.getMessage()).detail(VerificationError.ExceptionAttribute.EXCEPTION_CLASS, e.getClass().getName())
```

Comment:
Same.

Reply:
My "mvn clean package install -Psourcecheck" does not complain about this:

> [INFO] --- maven-checkstyle-plugin:3.0.0:checkstyle (default-cli) @ camel-aws-ddb ---
> [INFO] Starting audit...
> Audit done.
> [INFO] 

Maybe some other checkstyle to check? (I could fix that manually, but I want to learn for future PR how to detect that)

Reply:
This is my fault, since this is a builder, there shouldn't be any warning.